app/controllers/admin/volunteers.py

```diff
@@ -92,13 +105,58 @@ def addVolunteer(volunteer, eventId):
         flash("Error when adding volunteer", "danger")
     return ""
 
+@admin_bp.route('/addOutsideParticipantToEvent/<outsideParticipant>/<eventId>', methods = ['POST'])
+def addOutsideParticipant(outsideParticipant, eventId):
+    email = outsideParticipant.strip("()").split('(')[-1]
```

Comment:
Passing this info in the url is a bit weird, when you are already posting.

Reply:
I have fixed that.

Reply:
I don't see what is fixed. You are still passing that in the url instead of the post request